src/components/OrderList/OrderList.js

```diff
@@ -28,17 +28,19 @@ function OrderList({ filter }) {
 
   const displayOrders = () => {
     let isEmpty = true;
-    const orderView = userOrders?.map((order, index) => {
+    const orderView = userOrders?.map((order) => {
       const { orderStatus } = order;
       if (filter === "all" || filter === orderStatus.toLowerCase()) {
         isEmpty = false;
         return (
-          <tr>
+          <tr
+            style={{ cursor: "pointer" }}
+            onClick={() => history.push(`/admin/orders/${order.simplifiedID}`)}
+          >
```

Comment:
Nice! This kind of subtle implementation can go a long way when it comes to improving UX 💯